manifests/u/UltraDefragDevelopmentTeam/UltraDefrag/7.1.4/UltraDefragDevelopmentTeam.UltraDefrag.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: UltraDefragDevelopmentTeam.UltraDefrag
+PackageVersion: 7.1.4
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
This installer can only install on x64 unless you also add the x86 installer.

---

Huh, I though wingetreate could detect the installer architecture.  It did on other programs so I thought it would do the same here.